### ISSUE TITLE
ui5-combobox focus method not working

### ISSUE THREAD

**Issue by ee92** [SAP/ui5-webcomponents#2895]
**Describe the bug**
Using the `.focus()` method of the ui5-combobox does not work as expected. After calling `.focus()`, the ui5-combobox does not become the `document.activeElement`, does not have the focus outline, and has the `.focused` property set to false.

**To reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/ComboBox/
2. Using the dev tools, save a ui5-combobox as a variable
3. Call `.focus()` on it and notice it does not get focused as expected.

**Expected behavior**
When `.focus()` is called on the ui5-combobox, it should become the `document.activeElement`, should have a focus outline, and should have the property `.focused` set to true.

**Screenshots**
![Screen Shot 2021-03-01 at 9 25 18 AM](https://user-images.githubusercontent.com/24360164/109535013-e1506400-7a70-11eb-9f82-67a93194b526.png)

**Context**
- UI5 Web Components version: 0.29.0

**Affected components**
ui5-combobox

**Organization:**
SuccessFactors

**Priority:** *(optional)*
Medium

Thank you for looking into it :)

**Comment by terezamch** [comment]
Hello coleagues,

There is a missing data-sap-focus-ref attribute and therefore the component is not focused.

@ee92 , as for the outline - it is not rendered as the focus is in the console while executing the .focus() line.
It should work fine in the app logic.

Thanks,
Tereza

**Comment by ilhan007** [comment]
FYI @SAP/ui5-webcomponents-topic-rl this issue has been assigned to you
